bodhi/tests/client/test___init__.py

```diff
@@ -392,6 +392,6 @@ def test_url_flag(self, send_request, query):
                 'close_bugs': True, 'stable_karma': None, 'csrf_token': 'a_csrf_token',
                 'staging': False, 'builds': u'nodejs-grunt-wrap-0.3.0-2.fc25', 'autokarma': True,
                 'edited': u'nodejs-grunt-wrap-0.3.0-2.fc25', 'suggest': None,
-                'notes': u'this is an edited note', 'request': None, 'bugs': u'',
-                'unstable_karma': None, 'type': 'bugfix'})
+                'notes': u'this is an edited note', 'notes_file': None,
+                'request': None, 'bugs': u'', 'unstable_karma': None, 'type': 'bugfix'})
         self.assertEqual(bindings_client.base_url, 'http://localhost:6543/')
```

Comment:
It could also be good to add a couple of true unit tests in here on the new ```_process_notes_file()``` function. You could pass in dictionaries of various forms to exercise the various if statements within the function. Here are some ideas:

1. Assert that not passing ```--notes-file``` doesn't do anything.
2. Assert that passing ```--notes-file``` works in the normal case.
3. Assert that passing ```--notes-file``` and ```--notes``` makes it angry.

Unit tests like this can actually be pretty fun to write. Let me know if you need help getting started on these.